src/puppetlabs/puppetdb/anonymizer.clj

```diff
@@ -109,6 +110,12 @@
       (rule-match? x context) (get x "anonymize")
       :else (recur (rest xs)))))
 
+(defn anonymize-text
+  "This is for anonymizing text data where we care only about size, not content."
+  [text]
+  (when text
+    (StringUtils/repeat "?" (count text))))
```

Comment:
Why not just use `clojure.string` instead of adding a dependency? `(clojure.string/join (repeat (count text) "?")`